draft-ietf-core-senml-data-ct.md

```diff
@@ -177,36 +183,39 @@ When a SenML Record contains a Data Value field ("vd"), the Record MAY
 also include a Content-Format indication field, using label "ct".  The
 value of this field is a Content-Format-Spec, i.e., one of:
 
-* a CoAP Content-Format identifier in decimal form with no leading
+* a CoAP Content-Format number in decimal form with no leading
   zeros (except for the value "0" itself).  This value represents an
   unsigned integer in the range of 0-65535, similar to the CoRE Link
   Format {{?RFC6690}} "ct" attribute).
 
 * or a Content-Format-String containing a Content-Type and
-  optionally a Content-Coding (see below).
+  optionally zero or more content codings (see below).
```

Comment:
"optionally" and "zero or more" might be redundant (vs "optionally one or more")